packages/rest-hooks/src/state/selectors/useDenormalized.ts

```diff
@@ -63,13 +63,31 @@ export default function useDenormalized<
     }
 
     // second argument is false if any entities are missing
-    const [denormalized, entitiesFound]: [any, boolean] = denormalize(
+    const [denormalized, entitiesFound, cache] = denormalizeAndTracked(
       results,
       schema,
       entities,
     );
-    return [denormalized, entitiesFound] as any;
+
+    // this enables us to keep referential equality based on entities contained within
+    const entitiesList = Object.values(cache)
+      .map(Object.values)
+      .reduce((a: any[], b: any[]) => a.concat(b), [])
+      .join(',');
```

Comment:
serializing this list so react doesn't complain. this is somewhat dangerous as people not using `Entity` will have troubles.